### DISCUSSION TITLE
Is it possible to find all static jumps to a specific address or address range?

### DISCUSSION THREAD

**Discussion by TommyJerryMairo** [NationalSecurityAgency/ghidra#3929] [Q&A] [answered]
For example, the `jmp` instruction with hardcoded address or `rip` related address for PIEs, as well as not directly hardcoded for jump but loaded into a register which then used for indirect jumps without modifying or with unknown value independent modify (like modifies that could be determined statically).

**Comment by bdemick** [comment, marked as answer]
You can write a PCode analysis script to analyze `BRANCH` and `BRANCHIND` instructions to do this in an architecture-independent way. Not 100% sure if indirect targets are always fully resolved at the raw PCode level, so you might need high-level PCode in some cases.
